src/index.html

```diff
@@ -5,10 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="./images/logo.png" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+            href="#laptops&computers"class="nav_link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_moyo-header/assets/77551820/ba34211d-42c6-46af-a407-3744d606a4a3">

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
+